src/components/interfaces/MOBILE_API.xml

```diff
@@ -4095,6 +4096,9 @@
         <param name="steeringWheelAngle" type="Boolean" mandatory="false">
             <description>Current angle of the steering wheel (in deg)</description>
         </param>
+        <param name="engineOilLife" type="Boolean" mandatory="false">
```

Comment:
parameter "engineOilLife" shall be listed under GetVehicleData/OnVehicleData/SubscribeVehicleData/UnsubscribeVehicleData rpc in group "VehicleInfo-3" in file sdl_preloaded_pt.json

Reply:
@yang1070 fixed in https://github.com/smartdevicelink/sdl_core/pull/1823/commits/81f0253255e79b491f682f32cc961baacd64452e